File: AD-HOC/2444.js
let [V, T] = lines.shift().split(' ').map(Number);
let volumes = lines.shift().split(' ').map(Number);

let final = V;

for (let i = 0; i < T; i++) {
  final += volumes[i];

  if (final > 100) {
    final = 100;
  } else if (final < 0) {
    final = 0;
  }
}

console.log(final);

File: AD-HOC/2444.py
entrada = list(map(int, input().split()))

V = entrada[0]
T = entrada[1]

volumes = list(map(int, input().split()))

final = V

for i in range(T):
  final += volumes[i]

  if (final > 100):
    final = 100
  elif (final < 0):
    final = 0

print(final)
